src/barsPlus-props.js

```diff
@@ -364,21 +365,19 @@ const definition ={
         },
         measureAxis: {
           type: "items",
-          label: function (data) {
+          label: function (data, handler) {
             var t = "";
             if (data.qHyperCubeDef.qDimensions.length == 0 || data.qHyperCubeDef.qMeasures.length > 1) {
               if (data.props.axisTitleM.length > 0)
                 t = " (" + data.props.axisTitleM + ")";
             }
             else if (data.qHyperCubeDef.qMeasures.length > 0) {
-              const { qDef } = data.qHyperCubeDef.qMeasures[0];
-              if (qDef.hasOwnProperty("qLabel") && qDef.qLabel) {
-                t = qDef.qLabel;
+              var cId = data.qHyperCubeDef.qMeasures[0].qDef.cId;
+              var mesLayout = handler.getMeasureLayout(cId);
+              if (mesLayout !== undefined){
+                t = mesLayout.qFallbackTitle;
+                t = " (" + (t.startsWith("=") ? t.slice(1) : t) + ")";
```

Comment:
This looks like a more universal method. Is this something we use in other places as well?

Reply:
☝️ @ahmed-Bazzara

Reply:
Also this is completely optional: use template literals for better readability